templates/default/footer.tpl.html

```diff
@@ -1,3 +1,9 @@
 <footer>
-    <a href="mailto:<%= contact %>"><%=contact%></a>
-</footer>
\ No newline at end of file
+        <div>
+                <%= town %>
+                <%= address %>
+            </div>
```

Comment:
Fix indentation for above lines. Use standard format for street address (street name above city - and something to separate them). You seem to have forgotten to check what this looks like in a browser